src/main/scala/dpla/ingestion3/mappers/providers/HathiMapping.scala

```diff
@@ -0,0 +1,703 @@
+    "CS" -> (Some("Software"), Some("Software")),
+    "DI" -> (Some("Dictionaries"), Some("Text")),
+    "DR" -> (Some("Directories"), Some("Text")),
+    "EN" -> (Some("Encyclopedias"), Some("Text")),
+    "HT" -> (Some("HathiTrust"), None),
+    "MN" -> (Some("Maps-Atlas"), Some("Image")),
+    "MP" -> (Some("Map"), Some("Image")),
+    "MS" -> (Some("Musical Score"), Some("Text")),
+    "MU" -> (Some("Music"), Some("Text")),
+    "MV" -> (Some("Archive"), Some("Collection")),
+    "MW" -> (Some("Manuscript"), Some("Text")),
+    "MX" -> (Some("Mixed Material"), Some("Collection")),
+    "PP" -> (Some("Photograph/Pictorial Works"), Some("Image")),
+    "RC" -> (Some("Audio CD"), Some("Sound")),
+    "RL" -> (Some("Audio LP"), Some("Sound")),
+    "RM" -> (Some("Music"), Some("Sound")),
+    "RS" -> (Some("Spoken word"), Some("Sound")),
+    "RU" -> (None, Some("Sound")),
+    "SE" -> (Some("Serial"), Some("Text")),
+    "SX" -> (Some("Serial"), Some("Text")),
+    "VB" -> (Some("Video (Blu-ray)"), Some("Moving Image")),
+    "VD" -> (Some("Video (DVD)"), Some("Moving Image")),
+    "VG" -> (Some("Video Games"), Some("Moving Image")),
+    "VH" -> (Some("Video (VHS)"), Some("Moving Image")),
+    "VL" -> (Some("Motion Picture"), Some("Moving Image")),
+    "VM" -> (Some("Visual Material"), Some("Image")),
+    "WM" -> (Some("Microform"), Some("Text")),
+    "XC" -> (Some("Conference"), Some("Text")),
+    "XS" -> (Some("Statistics"), Some("Text"))
+  )
+
+  private val dataProviderMapping: Map[String, String] = Map(
+    "bc" -> "Boston College",
+    "chi" -> "University of Chicago",
+    "coo" -> "Cornell University",
+    "dul1" -> "Duke University",
+    "gri" -> "Getty Research Institute",
+    "hvd" -> "Harvard University",
+    "ien" -> "Northwestern University",
+    "inu" -> "Indiana University",
+    "loc" -> "Library of Congress",
+    "mdl" -> "Minnesota Digital Library",
+    "mdp" -> "University of Michigan",
+    "miua" -> "University of Michigan",
+    "miun" -> "University of Michigan",
+    "nc01" -> "University of North Carolina",
+    "ncs1" -> "North Carolina State University",
+    "njp" -> "Princeton University",
+    "nnc1" -> "Columbia University",
+    "nnc2" -> "Columbia University",
+    "nyp" -> "New York Public Library",
+    "psia" -> "Penn State University",
+    "pst" -> "Penn State University",
+    "pur1" -> "Purdue University",
+    "pur2" -> "Purdue University",
+    "uc1" -> "University of California",
+    "uc2" -> "University of California",
+    "ucm" -> "Universidad Complutense de Madrid",
+    "ufl1" -> "University of Florida",
+    "uiug" -> "University of Illinois",
+    "uiuo" -> "University of Illinois",
+    "umn" -> "University of Minnesota",
+    "usu" -> "Utah State University Press",
+    "uva" -> "University of Virginia",
+    "wu" -> "University of Wisconsin",
+    "yale" -> "Yale University"
+  )
+
+  private val rightsMapping: Map[String, String] = Map(
```

Comment:
Seems like there's a pretty easy rightsstatements implementation here. Do most of the values in their plain-text rights statements map to one of these?

Reply:
Yes, Scott and I made a ticket for this - Scott suggested that we should check with Hathi first to see what version of the licenses they are using.